### ISSUE TITLE
dont use c.URL_BASE

### ISSUE THREAD

**Issue by binary1230** [magfest/mivs#4]
we should use relative linking instead if it's possible, instead of using c.URL_BASE

we do this currently in mivs/templates/mivs_judging/index.html

see disccussion on https://github.com/magfest/mivs/pull/3/files and https://github.com/magfest/ubersystem/issues/1534

**Comment by EliAndrewC** [comment]
I used the full URL since the purpose of that link is to allow an admin to right-click and select "Copy link address" and then get a full URL that they could copy into an email to the studio.

I noticed that Google Chrome will copy the full URL even if you select "Copy link address" on a relative link, but I wasn't sure if this was also the behavior of other browsers.  (I don't even have an Internet Explorer to test on.)

If it is the case that relative links will copy into fully qualified links then we can definitely remove the ``c.URL_BASE`` from this page.
